rs/node_rewards/rewards_calculation/src/performance_based_algorithm/results.rs

```diff
@@ -5,71 +5,119 @@ use rust_decimal::Decimal;
 use serde::{Deserialize, Serialize};
 use std::collections::BTreeMap;
 
-pub type XDRPermyriad = Decimal;
+pub type XDRPermyriad = Decimal; // Rewards unit in XDR scaled by 1/10,000 (permyriad)
 pub type Percent = Decimal;
 pub type Region = String;
 
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq)]
 pub struct NodeMetricsDaily {
+    /// The subnet to which this node was assigned on this day
     pub subnet_assigned: SubnetId,
+    /// Subnet Assigned Failure Rate
+    ///
+    /// The failure rate of the entire subnet for this day.
     pub subnet_assigned_fr: Percent,
+    /// Number of blocks proposed by this node on this day
     pub num_blocks_proposed: u64,
+    /// Number of blocks that failed to be included on this day
     pub num_blocks_failed: u64,
+    /// Original Failure Rate
+    ///
+    /// Calculated as `num_blocks_failed / (num_blocks_proposed + num_blocks_failed)`.
+    /// Represents the raw failure rate of the node before any subnet-level adjustments.
     pub original_fr: Percent,
+    /// Relative Failure Rate
+    ///
+    /// Failure rate adjusted for subnet performance.
+    /// Calculated as `max(0, original_fr - subnet_assigned_fr)`.
     pub relative_fr: Percent,
 }
 
 #[derive(Serialize, Deserialize, Debug, PartialEq, Clone)]
 pub enum NodeStatus {
+    /// Node is assigned to a subnet with recorded metrics
     Assigned { node_metrics: NodeMetricsDaily },
-    Unassigned { extrapolated_fr: Percent },
+    /// Node is unassigned; only extrapolated failure rate is available
+    Unassigned {
+        /// Extrapolated Failure Rate (EFR)
+        /// Used to estimate the node's performance when unassigned
+        extrapolated_fr: Percent,
+    },
 }
 
 #[derive(Serialize, Deserialize, Debug, PartialEq, Clone)]
 pub struct NodeResults {
```

Comment:
A pre-existing issue.

"result" is super generic. Instead, how about `DailyNodeReward`, or something like that?